### DISCUSSION TITLE
Anyone tried Sensecap T1000A/ T1000B with Meshtastic?

### DISCUSSION THREAD

**Discussion by barryhunter** [meshtastic/Discussions#139] [nRF52]
Just curious if the  T-1000-A or  T-1000-B variants work with Meshtastic? 

I understand the T1000-E was specifically developed for Meshtastic so that it has a compatible GPS. (I think as meshtastic currently requires UART GPS) 

But would the A/B variants work, without GPS. Ie meshtastic would still work as a node?

Asking because in some ways seem that the Sensecap devices would make a reasonable remote repeater. ie they have compact & waterproof enclosure, and nice battery included. They dont waste any battery on screens, wifi etc. Not even a charging LED :)

And for a repeater/router, GPS is not important anyway. The B varient is slightly cheaper than E. - So could use a A/B as a simple router.

**Comment by GUVWAF** [comment]
The T1000A and B are not based on nRF52. I'm not sure what it uses, but it's not an MCU Meshtastic supports.

**Comment by caveman99** [comment]
T1000A, B, C and D are based on a hardware platform that is not compatible with mestastic and for lack of ressources will never be. They only work on lorawan with the provided firmware. The entire device only shares the enclosure with the T1000E and is otherwise completely different. T1000S shares the platform with T1000E but lacks an external USB port. Also it was never in wider circulation and only samples exist.

**Comment by barryhunter** [comment]
Thanks, that good info. 

Didnt realize the MCU is different, noticed the lack of details about the MCU in the datasheet, but just assumed the range would all use the same processor. 

Will drop the idea of using a 'B' as a cheap repeater :)
